homeassistant/components/vicare/sensor.py

```diff
@@ -477,6 +478,15 @@ class ViCareSensorEntityDescription(SensorEntityDescription, ViCareRequiredKeysM
         device_class=SensorDeviceClass.TEMPERATURE,
         state_class=SensorStateClass.MEASUREMENT,
     ),
+    ViCareSensorEntityDescription(
+        key="volumetric_flow",
+        name="Volumetric flow",
+        icon="mdi:gauge",
+        native_unit_of_measurement="l/h",
```

Comment:
I wonder if this should be added to [`UnitOfVolumeFlowRate`](https://github.com/home-assistant/core/blob/5ee62f29659922960f9572243e8d553a91e4ba84/homeassistant/const.py#L776C37-L776C37).

Reply:
not sure, since native unit is "liter" according the vicare api (which should be "liter/hour" offcourse) and UnitOfVolumeFlowRate doesn't support "l/h".